cypress/utils/commands.js

```diff
@@ -633,6 +643,60 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add(
+  'selectFromDataSourceSelectable',
+  (dataSourceTitle, dataSourceId) => {
+    cy.get('#dataSourceSelectableContextMenuPopover').click();
+    cy.checkDefaultDataSourceExist();
+    cy.getElementByTestId('dataSourceSelectable')
+      .find('input')
+      .clear('{backspace}')
+      .type(dataSourceTitle);
+    cy.wait(1000);
+    let dataSourceElement;
+    if (dataSourceId) {
+      dataSourceElement = cy.get(`#${dataSourceId}`);
+    } else if (dataSourceTitle) {
+      dataSourceElement = cy
+        .get('.euiSelectableListItem')
+        .contains(dataSourceTitle)
+        .closest('.euiSelectableListItem');
+    }
+    dataSourceElement.click();
+    // Close data source selectable manually if no data source element need to be clicked
+    if (!dataSourceElement) {
+      cy.getElementByTestId('dataSourceSelectable').last('button').click();
+    }
+  }
+);
+
+Cypress.Commands.add('checkDefaultDataSourceExist', () => {
+  cy.contains('li.euiSelectableListItem', 'default-ds')
```

Comment:
should we remove "default-ds", and add a title param to pass in here?

Reply:
I made a constant and updated